Python/Subdomain_Finder/README.md

```diff
@@ -0,0 +1,38 @@
+### Subdomain Finder
+
+This Script is used to find the possible subdomains of a domain.
+
+### Domain name
+
+The url format for a domain name is: "http://subdomain.domain.top-level-domain"
```

Comment:
This doesn't look right, for a starter keep this in either italics or backticks (the URL I mean)